src/applications/tab-selection/ClassicTabSelectionFormApplication.svelte.ts

```diff
@@ -75,11 +106,11 @@ export default class ClassicTabSelectionFormApplication extends SvelteFormApplic
     return [];
   }
 
-  createComponent(node: HTMLElement): Record<string, any> {
+  _createComponent(node: HTMLElement): Record<string, any> {
     return mount(TabSelection, {
       target: node,
       context: new Map<any, any>([
-        ['context', this.context],
+        ['context', this._context.data],
```

Comment:
Consider going back to the state rune approach instead of the coarse reactivity provider :/